1ejer/index.css

```
@@ -0,0 +1,111 @@
+.container{
+    max-width: 1200px;
+    margin: auto;
+    display: flex;
+    gap: 24px;
+    padding: 20px;
+}
+
+.left{      
+    width: 40%;
+    margin-top: 69px;
+    margin-bottom: 90px;
+    display: flex;
+    flex-direction: column;
+}
+
+.title{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 54px;
+    text-align: center;
+    color: #000000;
+    margin-top: auto;
+}
+
+.reference{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 18px;
+    line-height: 1.33;
+    letter-spacing: normal;
+    text-align: center;
+    color: #000000;
+}
+
+.description{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 12px;
+    line-height: 1.25;
+    letter-spacing: normal;
+    text-align: center;
+    color: #000000;
+}
+
+button{
```

Mejor ponerle algún selector más así no se aplica a todos los botones de la página.

```suggestion
.ejer1-container button {
```